templates/blog/product-cards.html

```diff
@@ -26,6 +26,16 @@ <h3>
   <p>Kubernetes, or K8s for short, is an open source platform pioneered by Google, which started as a simple container orchestration tool but has grown into a platform for deploying, monitoring and managing apps and services across clouds.</p>
   <p><a href="/kubernetes/what-is-kubernetes">Learn more about Kubernetes &rsaquo;</a></p>
 </div>
+{% elif article.tags and 3680 in article.tags %}
+<div class="p-card" id="rtp-bosch">
+<img alt="IoT infographic" src="https://assets.ubuntu.com/v1/53163a88-IOT_core_infographic.svg" class="">
```

Comment:
Shouldn't the class be `p-card__thumbnail`

Reply:
Tried, but `p-card__thumbnail` is for logos apparently, it makes things tiny. Eg.
![Screenshot from 2020-08-05 15-00-46](https://user-images.githubusercontent.com/18480003/89427831-fa43a180-d73b-11ea-9269-cd7a37bd315a.png)

Reply:
Can you remove the empty class attr then please?